spec/system/planning_applications/post_validation_requests_spec.rb

```diff
@@ -17,13 +16,110 @@
       create :planning_application, :in_assessment, local_authority: default_local_authority
     end
 
-    it "displays the post validation request table" do
```

Comment:
Because the file name didn't have `_spec` at the end this file wasn't being run, so this test was failing silently because of a css typo 😆 . However I think we've now got so many tests covering this table that we probably don't need this one, which just checks that the user can follow a link to the page.